action/action-util/src/main/java/com/powsybl/action/util/LoadFlowBasedPhaseShifterOptimizerConfig.java

```diff
@@ -24,22 +24,22 @@ public static LoadFlowBasedPhaseShifterOptimizerConfig load() {
     public static LoadFlowBasedPhaseShifterOptimizerConfig load(PlatformConfig platformConfig) {
         Objects.requireNonNull(platformConfig);
         ModuleConfig config = platformConfig.getModuleConfig("load-flow-based-phase-shifter-optimizer");
-        Class<? extends LoadFlowFactory> loadFlowFactoryClass = config.getClassProperty("load-flow-factory", LoadFlowFactory.class);
-        return new LoadFlowBasedPhaseShifterOptimizerConfig(loadFlowFactoryClass);
+        String loadFlowName = config.getOptionalStringProperty("load-flow-name").orElse(null);
```

Comment:
Don't forget to update this [page](http://www.powsybl.org/docs/configuration/modules/load-flow-based-phase-shifter-optimizer.html)

Reply:
Done